ext/opentelemetry-ext-pymemcache/src/opentelemetry/ext/pymemcache/__init__.py

```diff
@@ -0,0 +1,176 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+
+Usage
+-----
+
+The OpenTelemetry ``pymemcache`` integration traces pymemcache client operations
+
+Usage
+-----
+
+.. code-block:: python
+
+    from opentelemetry import trace
+    from opentelemetry.sdk.trace import TracerProvider
+    from opentelemetry.ext.pymemcache import PymemcacheInstrumentor
+    trace.set_tracer_provider(TracerProvider())
+    PymemcacheInstrumentor().instrument()
+    from pymemcache.client.base import Client
```

Comment:
Might be good to add a comment that `instrument` must be called before `Client` is imported.

Reply:
is that actually necessary? if only the methods on Client are being wrapped (and not Client itself) it should be fine, right?

Reply:
Ahh right, I thought this had the same implications as `Flask` when we patched the library itself.